src/cli/commands/object/links.js

```diff
@@ -21,25 +20,16 @@ module.exports = Command.extend({
       if (err) {
         throw err
       }
-      const mh = utils.isDaemonOn()
-        ? key
-        : new Buffer(bs58.decode(key))
 
-      ipfs.object.links(mh, (err, links) => {
+      ipfs.object.links(key, {enc: 'base58'}, (err, links) => {
         if (err) {
           log.error(err)
           throw err
         }
 
-        if (links.Links) { // js-ipfs-api output
-          links.Links.forEach((link) => {
-            console.log(link.Hash, link.Size, link.Name)
-          })
-          return
-        }
-
         links.forEach((link) => {
-          console.log(bs58.encode(link.hash).toString(), link.size, link.name)
+          link = link.toJSON()
```

Comment:
yes there is, `.Hash`, `.Size` and `.Name` are not available otherwise is this nice format